packages/websocket-plugin/src/symbols.ts

```diff
@@ -74,6 +74,16 @@ export class DisconnectWebSocket {
   }
 }
 
+/**
+ * Action triggered when websocket is disconnected
+ */
+export class WebSocketDisconnected {
```

Comment:
`WebsocketDisconnected`

Reply:
existing actions mostly have capital S 
```
ConnectWebSocket,
DisconnectWebSocket,
SendWebSocketMessage,
WebsocketMessageError

Reply:
@markwhitfeld is it worth renaming `WebsocketMessageError` to `WebSocketMessageError` thus all actions will have capital S?

Reply:
Well, considering that this action was not working before this PR, I don't think it will be breaking anything which wasn't broken. Happy to change

Reply:
@arturovt I think it should be changed, but an alias needs to be added for backward compatibility until the next major version.

Reply:
@eranshmil I would have agreed with you on backward compatibility, but WebsocketMessageError was not being dispatched before this PR. It won't be a breaking change to change the broken.

Reply:
@kwalski rename for consistency with little S everywhere

Reply:
@splincode rename with 's' for all actions?! That would be breaking change??

Reply:
Ooh, yes

But, WebSocketMessageError also a critical change since you renamed